.gitignore

```diff
@@ -9,6 +9,7 @@ Gemfile.lock
 gemfiles/*.lock
 InstalledFiles
 .idea/
+*.iml
```

Comment:
Any idea what this is?

Reply:
This is for blacklisting the Intellij IDEA IDE cache file